### ISSUE TITLE
Empty navigation means should be unavailable

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#2724]
The Navigation panel starts with a dropdown menu to select a navigation mean (TOC, landmark, gotopage, annotation, bookmark). When one of those means have no entry, it is still reachable and present an empty panel. 

To prevent users from searching to understand where they are, when a navigation mean is empty, it should not be reachable. 

![screenshot showing an empty landmark navigation](https://github.com/user-attachments/assets/861c542a-3d37-4837-bbb0-ba0579860cff)

**Comment by gautierchomel** [comment]
Note that this is the case when that panel is not docked 
![image](https://github.com/user-attachments/assets/2788711d-50fc-4dea-8a6b-0a1a2dbdf2dd)
